### ISSUE TITLE
Interaction commands inconsistent about running unfocussing steps

### ISSUE THREAD

**Issue by andreastt** [w3c/webdriver#1212]
The Element Clear commands runs the unfocussing steps after having cleared the element, causing the `blur` event to fire and the focussed element to be changed to the element’s controlling element, but the Element Send Keys command does not.

I would expect the high-level commands to behave uniformly and return focus to its controlling element or to where focus was at before the invocation of the command. As a corollary I would _not expect_ the low-level action commands to unfocus the element.

**Comment by andreastt** [comment]
@AutomatedTester Thoughts?

**Comment by andreastt** [comment]
cc @shs96c

**Comment by andreastt** [comment]
Ping @AutomatedTester

**Comment by andreastt** [comment]
Ping again @AutomatedTester or @shs96c.

**Comment by AutomatedTester** [comment]
@andreastt I have the same expectations as you on this one

**Comment by shs96c** [comment]
The original behaviour in selenium was based on:

`Element Clear` was typically used to blank a field ready for user inputs. Most JS waited for the `onchange` event to fire before running validations. As a result, this command needed to fire the event, or else people's expectations weren't being met. There were a surprising number of complaints about the event _not_ firing when it didn't. *sigh*

`Element Send Keys`, however, was designed to support method chaining (that is, someone might have code that looked like `element.sendKeys("abc"); element.sendKeys(SOME_CONSTANT);`, where `SOME_CONSTANT` is another method call) This is one of the reasons that the parameter for `sendKeys` used to be an array. Because people were dipping in and out of the remote end as they made the method calls, it wasn't appropriate to fire the `onchange` event.

So, that's the history of why these commands behave the way that they do. As we wrote the spec, we maintained the expected behaviour.
